### PR TITLE
fix: use chain for TransformableStream to fail if TransformationException

### PR DESCRIPTION
**Pull request by ytvnr**
**Issue**

https://github.com/gravitee-io/issues/issues/7130

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.6.1-SNAPSHOT.issues-7130-transformationexception-chain`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-json-to-json/1.6.1-SNAPSHOT.issues-7130-transformationexception-chain/gravitee-policy-json-to-json-1.6.1-SNAPSHOT.issues-7130-transformationexception-chain.zip)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
     </parent>
 
     <properties>
+        <gravitee-bom.version>1.4</gravitee-bom.version>
```
Comment (gaetanmaisse) [inline pom.xml:37]:
```suggestion
        <gravitee-bom.version>2.3</gravitee-bom.version>
```

Comment (gaetanmaisse) [reply pom.xml:37]: We keep 1.x for compatibility with APIM 3.10 and will move to 2.x for APIM 3.15

**Review by gaetanmaisse — APPROVED** [review]

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**